### DISCUSSION TITLE
Markdown renderer now puts header underline over images

### DISCUSSION THREAD

**Discussion by nico-bohlinger** [community/community#111402] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello everyone,

I saw that the GitHub markdown renderer now puts the underline created by markdown headers (# Header) over images. This was previously not the case and has changed 2 weeks ago I think. The renderer used for the GitHub mobile app (at least on Android) still puts the image above the underline.
I personally prefer letting the image take priority such that neat usages of images in the readmes of some repositories still look great.
You can see such a case in the readme of this repository: https://github.com/DLR-RM/stable-baselines3

**Comment by GalaxyGaming2000** [comment]
Seems like a bug mobile isnt updated as much so that would explain it, have you submitted a bug report?

in the meantime you can move the image into the header area so the image will be below the line instead of having a line inside of it

**Comment by gdmcdonald** [comment]
Unfortunately this affects our .github profile as well: [Sydney Informatics Hub](https://github.com/Sydney-Informatics-Hub)

Would be great to not have the border over the image, like it was earlier this year.

![Screenshot 2024-10-15 at 6 29 41 PM](https://github.com/user-attachments/assets/ac0bd70b-4716-4c3d-8666-f38c1f0ec428)
